docs/_src/language/nesting.md

```diff
@@ -1,8 +1,8 @@
-# What is a "Turtle?"
+# What is an "Aggregating Subquery?"
 
-Turtles are queries nested in other queries. The technical name for what a turtle is doing is "aggregating subgquery--in Malloy, it is an object which has a name, and transforms a shape. "Aggregating subquery" is a bit of a mouthful, so we call it a turtle. The word comes from the philosophical phrase [Turtles All The Way Down](https://en.wikipedia.org/wiki/Turtles_all_the_way_down).
+Aggregating Subqueries are queries nested in other queries. In Malloy, it is an object which has a name, and transforms a shape. "Aggregating subquery" is a bit of a mouthful, so we often refer to it as "nesting a query."
```

Comment:
.. nested in other queries.  A nested query produces a sub-table per row in the query it is embedded in.